sites/fast-website/src/app/data/feature.data.ts

```diff
@@ -1,65 +1,133 @@
+interface FeatureLink {
+    anchorText: string;
+    url: string;
+}
+
 export interface FeatureCardData {
-    body: string;
-    documentationLink: string;
-    githubLink: string;
     header: string;
+    body: string;
+    links?: FeatureLink[];
 }
 
 export const featureCardData: FeatureCardData[] = [
     {
+        header: "Composition",
         body:
             "Create new component compositions by nesting, styling base components, and passing unhandled props. The combinations are endless.",
-        documentationLink: "#",
-        githubLink: "#",
-        header: "Composition",
+        links: [
+            {
+                anchorText: "View GitHub",
+                url: "#",
+            },
+            {
+                anchorText: "Read Documentation",
+                url: "#",
+            },
+        ],
     },
     {
+        header: "Design Systems",
         body:
             "Use Fluent by default or customize design system properties to make it your own. Or, create your own design system to use with FAST components.",
-        documentationLink: "#",
-        githubLink: "#",
-        header: "Design Systems",
+        links: [
+            {
+                anchorText: "View GitHub",
+                url: "#",
+            },
+            {
+                anchorText: "Read Documentation",
+                url: "#",
+            },
+        ],
     },
     {
+        header: "Technology",
         body:
             "Out of the box, FAST components are built on React, but you can build components on any modern framework using the FAST system.",
-        documentationLink: "#",
-        githubLink: "#",
-        header: "Technology",
+        links: [
+            {
+                anchorText: "View GitHub",
+                url: "#",
+            },
+            {
+                anchorText: "Read Documentation",
+                url: "#",
+            },
+        ],
     },
     {
+        header: "Web Standards",
         body:
             "All FAST components follow WCAG 2.1, are W3C spec-compliant and use the W3C interaction models when available.",
-        documentationLink: "#",
-        githubLink: "#",
-        header: "Web Standards",
+        links: [
+            {
+                anchorText: "WCAG",
+                url: "https://www.w3.org/WAI/standards-guidelines/wcag/",
+            },
+            {
+                anchorText: "Web Accessibility Initiative",
+                url: "https://www.w3.org/WAI/",
+            },
```

Comment:
Let's drop this link and just link to above

Reply:
Dropped